k8s/namespaces/jenkins/patches/ptlsbox/cluster-00/jenkins.yaml

```diff
@@ -5,8 +5,13 @@ metadata:
   name: jenkins
   namespace: jenkins
 spec:
+  chart:
+    repository: https://charts.jenkins.io
+    name: jenkins
+    version: 3.8.8
   values:
     controller:
+      tag: "dbhhxp"
```

Comment:
could you update the sidecar as well?
https://github.com/hmcts/shared-services-flux/blob/master/k8s/namespaces/jenkins/jenkins.yaml#L241

Reply:
done